### PR TITLE
[Applab Datasets] Implement getColumn

### PR DESCRIPTION
**Pull request by ajpal**
In toolbox:
image

In use:
image

Not done yet:
- Autopopulating dropdowns
- Doesn't work yet with empty columns:
image
Expected: `getColumn("mytable", "empty")` => `[undefined, undefined]`
Observed:
image

### CODE REVIEW

File: apps/src/applab/commands.js [hunk 1]
```diff
@@ -1829,6 +1829,45 @@ var handleSetKeyValueSyncError = function(opts, message) {
...
+    handleGetColumn.bind(this, opts),
+    handleGetColumnError.bind(this, opts)
+  );
+};
+
+var handleGetColumn = function(opts, records) {
+  let columnList = [];
+  if (records === null) {
+    outputError(
+      'You tried to read records from a table called ' +
```
Comment (jmkulwik) [inline apps/src/applab/commands.js]: Should these be i18n strings?

Comment (jmkulwik) [reply apps/src/applab/commands.js]: esp since they are repeated in line 1904

File: apps/src/applab/dropletConfig.js [hunk 2]
```diff
@@ -1099,6 +1100,17 @@ export var blocks = [
...
 ];
 
+if (experiments.isEnabled(experiments.APPLAB_DATASETS)) {
+  blocks.push({
+    func: 'getColumn',
+    parent: api,
+    category: 'Data',
+    paletteParams: ['table', 'column'],
+    params: ['"mytable"', '"mycolumn"'],
+    nativeIsAsync: true
```
Comment (jmkulwik) [inline apps/src/applab/dropletConfig.js]: Should this also be `type: 'value'` since it's a getter?

Comment (ajpal) [reply apps/src/applab/dropletConfig.js]: Good catch, thanks!

File: apps/src/applab/commands.js (hunk 1 above)
Comment (ajpal) [reply apps/src/applab/commands.js]: For some reason I had it in my head that we didn't support i18n error messages yet, but I think I was wrong. Fixed!

**Review by jmkulwik — APPROVED** [review]

**Pull request merged**